winpe.c

```diff
@@ -543,13 +550,15 @@ bool pe_parse(const uint8_t *const ptr, size_t const len, struct ParsedImage *im
             LOG("Misaligned raw data size");
             return false;
          }
+         // marmarek: is that really not allowed? could be necessary for alignment, no?
          if (image->sections[i].PointerToRawData != last_section_start) {
             LOG("Section %" PRIu32 " starts at 0x%" PRIx32 ", but %s at 0x%" PRIx32,
                 i, image->sections[i].PointerToRawData,
                 i > 0 ? "previous section ends" : "NT headers end",
                 last_section_start);
             return false;
          }
```

Comment:
If there is a gap between the sections, the PE file hashing algorithm won’t consider it.